test/Microsoft.AspNet.SignalR.Client.JS.Tests/wwwroot/Tests/FunctionalTests/Core/RedirectFacts.js

```diff
@@ -25,6 +25,120 @@ QUnit.asyncTimeoutTest("Can connect to endpoint which produces a redirect respon
     });
 });
 
+QUnit.asyncTimeoutTest("Can connect to endpoint which produces a redirect response with a query string", testUtilities.defaultTestTimeout, function (end, assert, testName) {
+    // "/redirect-query-string2" -> "/signalr?name1=newValue&name3=value3"
+    var connection = testUtilities.createTestConnection(testName, end, assert, { url: "redirect-query-string2", hub: true }),
+        proxies = connection.createHubProxies(),
```

Comment:
nit: formatting in this file, what's with the `,` instead of `;` and weird tabs?

Reply:
I'm not sure what's weird about the tabs. Back in the old SignalR days, we typically used the comma operator when possible when declaring multiple variable at the top of a functions. When in Rome...